docs/content/docs/guide/running.md

```diff
@@ -141,6 +141,11 @@ roses are red, violets are blue. pipeline are bound to flake by design.
 /test <pipelinerun-name>
 ```
 
+Note that the `/test <pipelinerun-name>` comment will start the `PipelineRun`
```

Comment:
Can we have a subsection specifying this part of the doc is to trigger not based on events something like that 

I mean

```
### GitOps Command Execution

#### Event-Based Triggers
....
(Describe GitOps commands triggered by specific events, such as pull requests or merges.)

#### Non-Event-Based Triggers
...
(Specify GitOps commands that are not triggered by specific events, providing details on how they can be initiated, e.g., through `/test` comments.)
```